packages/esm-patient-search-app/src/patient-search-bar/patient-search-bar.component.tsx

```diff
@@ -27,29 +29,41 @@ const PatientSearchBar = React.forwardRef<HTMLInputElement, React.PropsWithChild
     );
 
     const handleSubmit = useCallback(
-      (event: React.FormEvent<HTMLFormElement>) => {
+      (event) => {
         event.preventDefault();
         if (searchTerm && searchTerm.trim()) {
           onSubmit(searchTerm.trim());
+        } else {
+          setIsInputClicked(true);
+          inputRef.current.focus();
```

Comment:
Again, from my comments, I suggested only triggering this on a click on the button. Putting this is the `submit` handler also triggers when the user hits `enter` and that's exactly the thing we wanted to avoid.